.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,79 @@
+name: Python Benchmark
+
+on:
+  push:
+  pull_request:
+
+concurrency:
+  group: benchmark-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: ./bindings/python
+    shell: bash
+
+jobs:
+  build:
+    # supercharge/mongodb-github-action requires containers so we don't test other platforms
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["macos-latest"]
+        python-version: ["3.10"]
+      fail-fast: false
+    name: CPython ${{ matrix.python-version }}-${{ matrix.os }}
```

Comment:
The name doesn't need this metadata. If we remove the name it will show up as "Python Benchmark/build"

Reply:
Done.